lib/success.js

```diff
@@ -65,21 +65,62 @@ export default async function success(pluginConfig, context, { Octokit }) {
     const releaseInfos = releases.filter((release) => Boolean(release.name));
     const shas = commits.map(({ hash }) => hash);
 
-    const searchQueries = getSearchQueries(
-      `repo:${owner}/${repo}+type:pr+is:merged`,
-      shas,
-    ).map(
-      async (q) =>
-        (await octokit.request("GET /search/issues", { q })).data.items,
-    );
+    // MOCK ONLY
+    // const mockReponse = {
+    //   "data": {
+    //     "repository": {
+    //       "com1": {
+    //         "associatedPullRequests": {
+    //           "nodes": [
+    //             {
+    //               "number": 2
+    //             }
+    //           ]
+    //         }
+    //       },
+    //       "com2": {
+    //         "associatedPullRequests": {
+    //           "nodes": [
+    //             {
+    //               "number": 1
+    //             }
+    //           ]
+    //         }
+    //       }
+    //     }
+    //   }
+    // };
+    // console.log(somedata)
+
+    const { data } = await octokit.graphql(loadAssociatedPRsQuery(shas), { owner, repo });
+    
+    const associatedPRs = toArray(mockReponse.data.repository).map((item) => item.associatedPullRequests.nodes);
```

Comment:
is `toArray` doing the same as [`Object.values()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values) here?

Reply:
Yea, I found `toArray` to be more easy to read... Plus I didn't even think of `object.values`, as we've already established usage of `lodash`.

Reply:
In general, prefer native APIs over modules. We use lodash for legacy reasons but over time we might get rid of it as the need for it declines.

Reply:
True 😉